src/components/deploymentConfig/utils.tsx

```diff
@@ -68,9 +68,9 @@ export const makeObjectFromJsonPathArray = (index: number, paths: string[]) => {
 export const applyCompareDiffOfTempFormDataOnOriginalData = (
```

Comment:
Add comment on what this method does and also add can expect error since not wrapped in try catch